src/Promotions/FreeAddonModal/resources/App.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-
-import Modal from './Modal';
-
-const modalNode = document.createElement('div');
-document.body.append(modalNode);
-
-ReactDOM.render(<Modal />, modalNode);
```

Comment:
@JasonTheAdams if you're removing this file, you'll also need to remove it from the webpack config or else the bundle will fail.

Reply:
Good catch! Resolved in 2e8c176ace